arrows/ceres/register_algorithms.cxx

```diff
@@ -45,16 +44,27 @@ namespace kwiver {
 namespace arrows {
 namespace ceres {
 
-/// Register Ceres algorithm implementations with the given or global registrar
-int register_algorithms( vital::registrar &reg )
+extern "C"
+KWIVER_ALGO_CERES_EXPORT
+void
+register_factories( kwiver::vital::plugin_loader& vpm )
 {
-  REGISTRATION_INIT( reg );
+  static auto const module_name = std::string( "algorithms.ceres" );
+  if (vpm.is_module_loaded( module_name ) )
+  {
+    return;
+  }
 
-  REGISTER_TYPE( ceres::bundle_adjust );
-  REGISTER_TYPE( ceres::optimize_cameras );
```

Comment:
This is a new algorithm that needs to be register in the new plugin system.  The description is "Uses Ceres Solver to optimize camera parameters"